ironfish/src/network/peers/peerManager.ts

```diff
@@ -741,15 +741,6 @@ export class PeerManager {
     return !!peer.state.identity && this.banned.has(peer.state.identity)
   }
 
-  /**
-   * Send a message to all connected peers.
-   */
-  broadcast(message: NetworkMessage): void {
```

Comment:
This is trivial, just moving this logic to the `PeerNetwork` so that we can mark peers as having seen transaction hashes